### DISCUSSION TITLE
Processor example documentation

### DISCUSSION THREAD

**Discussion by muratisik-lab** [hneemann/Digital#1222] [Q&A]
Dear Mr Neemann,

I find the Digital tool very useful and well-designed for the Digital Design course I am teaching at university.

I see that in an earlier Q&A (#603) you have responded that documentation exists for the processor example only in German language.
Can we have access to the documentation in German? I am ready to comply with all the rules and regulations you provide.

Thank you
Murat Isik

**Comment by hneemann** [comment]
Maybe I should really write a short English documentation that briefly explains all the building blocks. 
Have you read the help texts, which are provided as tool-tips? Maybe it is better if these are extended, because they are always at hand.

**Comment by mace-de** [comment]
Dear Mr. Neemann,
is it possible to get the German description for the processor? I am from Germany and I would really like to read it for a better understanding of the processor architecture.
Anyway a huge thanks for making this great piece of software available. It's so interesting to play with it and have the possibility to export designs to real Hardware.
Regards Oliver
